tests/SprykerEcoTest/Zed/UnzerApi/Business/PerformAuthorizableChargeApiCallFacadeTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\UnzerApi\Business;
+
+use Generated\Shared\Transfer\UnzerApiChargeResponseTransfer;
+use Generated\Shared\Transfer\UnzerApiErrorResponseTransfer;
+
+/**
+ * @group Functional
+ * @group SprykerEco
+ * @group Zed
+ * @group UnzerApi
+ * @group Business
+ */
+class PerformAuthorizableChargeApiCallFacadeTest extends UnzerApiFacadeBaseTest
+{
+    /**
+     * @var string
+     */
+    protected const FIXTURE_FILE_NAME = 'chargeResponseBody.json';
+
+    /**
+     * @return void
+     */
+    public function testReturnsSuccessfulResponse(): void
+    {
+        // Arrange
+        $unzerApiRequestTransfer = $this->tester->createUnzerApiRequestTransfer();
+
+        // Act
+        $unzerApiResponseTransfer = $this->facade->performAuthorizableChargeApiCall($unzerApiRequestTransfer);
+        $unzerApiChargeResponseTransfer = $unzerApiResponseTransfer->getChargeResponseOrFail();
+
+        // Assert
+        $this->assertInstanceOf(UnzerApiChargeResponseTransfer::class, $unzerApiChargeResponseTransfer);
```

Comment:
this test is useless
there is no way to have something else than `UnzerApiChargeResponseTransfer`

same for similar cases across tests